libs/coin-evm/src/__tests__/unit/logic.unit.test.ts

```diff
@@ -398,8 +399,33 @@ describe("EVM Family", () => {
         setEnv("NFT_CURRENCIES", oldEnv);
       });
 
-      it("should provide a valid sha256 hash", () => {
-        expect(getSyncHash(currency)).toStrictEqual(expect.stringMatching(/^0x[A-Fa-f0-9]{64}$/));
+      beforeEach(() => {
+        // Clearing the cache of already calculated hashes before each new test
+        Object.keys(_simpleSyncHashMemoize).forEach(key => delete _simpleSyncHashMemoize[key]);
```

Comment:
IIUC the need to export `_simpleSyncHashMemoize` is just to be able to clear it here, right?

Unimportant but I'd rather export a function to clear it, rather then the variable itself. So it doesn't need the `_` prefix that triggers me :stuck_out_tongue: